operate/client/e2e-playwright/pages/Processes/Processes.ts

```diff
@@ -56,22 +54,20 @@ export class Processes {
     });
   }
 
-  async navigateToProcesses({
+  async gotoProcessesPage({
     searchParams,
     options,
   }: {
     searchParams?: Parameters<typeof convertToQueryString>[0];
     options?: Parameters<Page['goto']>[1];
   }) {
     if (searchParams === undefined) {
-      await this.page.goto(relativizePath(Paths.processes()));
+      await this.page.goto('/operate/processes');
```

Comment:
Before we were using the constants from the production code which is very bad since it might cause false positives. I'll try to generalize the URLs in the page objects to avoid hardcoding the strings in the tests